src/components/Course.js

```diff
@@ -29,37 +30,32 @@ class Course extends React.Component {
     mutator: PropTypes.object.isRequired,
   };
 
-  renderLastMenu = () => {
-    return (
-      <PaneMenu>
```

Comment:
I guess `<PaneMenu>` was never really needed?

Reply:
Ehhh, I think it probably was tbh. The `lastMenu` and `actionsMenu` have slightly different APIs I believe. `actionsMenu` renders the dropdown button itself so it doesn't need a "container" like `PaneMenu`.